stream_chat/async_chat/channel.py

```diff
@@ -148,17 +148,18 @@ async def truncate(self, **options):
         """
         return await self.client.post(f"{self.url}/truncate", data=options)
 
-    async def add_members(self, members, message=None):
+    async def add_members(self, members, message=None, options=None):
```

Comment:
how about `**options`?

Reply:
oh my bad, I didn't notice the pattern 🤦 fixed